string name
string app_name
app_scheduler/AppSchedule app_schedule
string[] app_args
